packages/web/tests/dotlottie.test.ts

```diff
@@ -550,6 +551,30 @@ describe.each([
     });
   });
 
+  describe('draw', () => {
```

Comment:
Nit:
Let’s remove this `describe` block since it doesn’t have any special setup or teardown for the renderError test